src/services/ksqldb/handlers/stageDdl.js

```diff
@@ -0,0 +1,70 @@
+import { send, SUCCESS, FAILED } from "cfn-response-async";
+var fs = require("fs");
```

Comment:
Change this to the following for consistency
```suggestion
import fs from "fs";
```